ui/components/institutional/confirm-remove-jwt-modal/confirm-remove-jwt-modal.js

```diff
@@ -0,0 +1,125 @@
+import React, { memo, useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import { useDispatch } from 'react-redux';
+import Modal from '../../app/modal';
+import CustodyAccountList from '../../../pages/institutional/connect-custody/account-list';
+import { useI18nContext } from '../../../hooks/useI18nContext';
+import { removeAccount } from '../../../store/actions';
+import withModalProps from '../../../helpers/higher-order-components/with-modal-props';
+import { Text } from '../../component-library';
+import Box from '../../ui/box';
+import {
+  BorderRadius,
+  DISPLAY,
+  TEXT_ALIGN,
+  TextColor,
+  TextVariant,
+} from '../../../helpers/constants/design-system';
+
+const ConfirmRemoveJWT = ({
+  custodyAccountDetails,
+  accounts,
+  token,
+  hideModal,
+}) => {
+  const t = useI18nContext();
+  const dispatch = useDispatch();
+  const [showMore, setShowMore] = useState(false);
+  const [tokenAccounts, setTokenAccounts] = useState([]);
+
+  useEffect(() => {
+    const lowercasedTokenAddress = token.address.toLowerCase();
+
+    const filteredAccounts = custodyAccountDetails.filter((item) => {
+      const addressLower = item.address.toLowerCase();
+      return accounts.find((acc) => acc.address.toLowerCase() === addressLower);
+    });
+
+    const tokens = filteredAccounts
+      .filter(({ authDetails }) => {
+        const getToken =
+          authDetails?.token ?? authDetails?.jwt ?? authDetails?.refreshToken;
+        return getToken?.toLowerCase() === lowercasedTokenAddress;
+      })
+      .map(({ address, name, labels, authDetails }) => {
+        const lowercasedAddress = address.toLowerCase();
+        const account = accounts.find(
+          ({ address: adressAcc }) =>
+            adressAcc.toLowerCase() === lowercasedAddress,
+        );
+        const balance = account?.balance;
+        const getToken =
+          authDetails?.token ?? authDetails?.jwt ?? authDetails?.refreshToken;
+        return { address, name, labels, balance, token: getToken };
+      });
+
+    setTokenAccounts(tokens);
+  }, [accounts, custodyAccountDetails, token]);
+
+  const handleRemove = async () => {
+    try {
+      for (const account of tokenAccounts) {
+        await dispatch(removeAccount(account.address.toLowerCase()));
+      }
+      hideModal();
+    } catch (error) {
+      console.error(error);
```

Comment:
Hey buddy, regarding the error handling, I'm not 100% sure about what is our standard and best practice here. But i can refer to some other components and we can add `  captureException(error);` to make it a bit more comprehensive, wdyt?
eg: `ui/ducks/institutional/institutional.js`,
`ui/ducks/alerts/unconnected-account.js` 

or we can `log` error, eg: `ui/ducks/swaps/swaps.js`
```
log.error(
        'Failed to fetch Swaps feature flags and Swaps liveness, defaulting to false.',
        error,
      );
```

Reply:
Ok, I'll talk with the team and if needed, we will create another PR. Thanks :)